exporter/datasetexporter/datasetexporter.go

```diff
@@ -38,11 +40,17 @@ func newDatasetExporter(entity string, config *Config, logger *zap.Logger) (*Dat
 			config.String(), err,
 		)
 	}
-
+	userAgent := fmt.Sprintf(
+		"%s;%s;%s",
+		"OtelCollector",
```

Comment:
Should we also include datasetexporter somewhere in this fragment? Or you think it's not needed since it will always be dataset exporter when the fragment also contains OtelCollector.